src/console/plugin.ts

```diff
@@ -191,5 +198,53 @@ function activateConsole(app: Application, services: ServiceManager, rendermime:
     text: 'Switch Kernel'
   }]);
 
+  let newSubmenu = new Menu ([
+    new MenuItem ({
+      text: 'Python 3 Console',
```

Comment:
We can't assume the user has a python 3 console.  This should use the display name of the "default" kernel which is the value of `services.kernelspecs.default`.